bucket/openOCD.json

```diff
@@ -1,10 +1,20 @@
 {
-    "version": "20201228",
+    "version": "0.11.0",
     "description": "OpenOCD is an open-source tool that allows debugging various ARM devices with GDB using a wide variety of JTAG programmers.",
-    "homepage": "https://gnutoolchains.com/arm-eabi/openocd/",
-    "license": "Unknown",
-    "url": "https://sysprogs.com/getfile/1364/openocd-20201228.7z",
-    "hash": "2a45b7fee2974c36650c5bae95daa9b577c2241557332818f08e27a364de4311",
-    "extract_dir": "OpenOCD-20201228-0.10.0",
-    "env_add_path": "bin"
+    "homepage": "https://openocd.org/",
+    "license": "GPL-2.0-or-later",
+    "url": "https://github.com/openocd-org/openocd/releases/download/v0.11.0/openocd-v0.11.0-i686-w64-mingw32.tar.gz",
+    "hash": "b195e3c3e9fe3f4fe9dfd9780ff08336ef8ed3015a76cc883d9e21e57c456f33",
+    "bin": "bin\\openocd.exe",
+    "checkver": {
+        "github": "https://github.com/openocd-org/openocd",
+        "regex": "/releases/tag/([\\w.]+)"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/openocd-org/openocd/releases/download/$version/win-portacle.exe#/dl.7z"
```

Comment:
Umm this won't work

Use https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate#using-checkverps1-to-query-and-autoupdate with ForceUpdate param and then re-install

Reply:
For offline testing.

There is no win-portacle.exe present.

This URL should match with version substitution the section above.

Reply:
Also this is 32 bit right?
It should be in the global block

Reply:
Hello! I've set autoupdate to github download url. Is this okay for this profile?

Reply:
@luojia65 that is not an issue.

Don't worry!!!

Create as many commits as you need to.